frontend/src/features/login/components/OtpForm.tsx

```diff
@@ -42,12 +42,8 @@ export const OtpForm = ({
   return (
     <form onSubmit={handleSubmit(onSubmitForm)}>
       <FormControl isInvalid={!!formState.errors.otp} mb="2.5rem">
-        <FormLabel
-          isRequired
-          htmlFor="otp"
-          description={`Enter OTP sent to ${email}`}
-        >
-          One-time password
+        <FormLabel isRequired htmlFor="otp">
+          {`Enter OTP sent to ${email.toLowerCase()}`}
         </FormLabel>
```

Comment:
actually this line should be localized too right haha

Reply:
Thanks! Actually not just this, the otp form as a whole is not localised 🤔 (only Login Form is localised)
I think expanding localisation is outside the scope of this PR though so I'll leave it for now 😅